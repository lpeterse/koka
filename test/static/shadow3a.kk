// shadow definition
function id(x) {
  fun id(y) = y;
  id(x)
}
