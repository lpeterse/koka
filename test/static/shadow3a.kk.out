id : forall<a> (x : a) -> a